lib/net.c

```diff
@@ -276,99 +389,50 @@ gatekeeper_setup_rss(uint8_t portid, uint16_t *queues, uint16_t num_queues)
 }
 
 static int
-init_port(struct net_config *net_conf, uint8_t port_id, uint8_t *num_succ_ports)
+init_port(struct gatekeeper_if *iface, uint8_t port_id, uint8_t *num_succ_ports)
```

Comment:
`num_succ_ports` should be renamed to `pnum_succ_ports` to avoid confusion in the code.